taf/plugins/pytest_sut_monitor.py

```diff
@@ -227,20 +229,23 @@ def list_rrd_folders(self):
         for dev_name in self.devices:
             collectd_folder = os.path.join(PATH_TO_RRD, dev_name)
             # List all RRD folder related to specific device
-            folders = self.exec_command('find {}/*'
-                ' -maxdepth 1 -type d -print0'.format(collectd_folder)).split('\0')
+            folders = self.exec_command(
+                'find {}/*' ' -maxdepth 1 -type d -print0'.format(collectd_folder)).split('\0')
```

Comment:
LinuxHostBash.cli_send_command() and paramiko exec_command() accept a string, not a list of strings.
List might be better but I don't want to change this in the whole TAF in this PR. We can change that in a separate PR.